pkg/helm/client.go

```diff
@@ -140,14 +140,25 @@ func (h CliClient) Install(ctx context.Context, chart, version, release, namespa
 	if wait {
 		cmd = append(cmd, "--wait")
 	}
-
-	out, err := RunCmdWithTimeout(ctx, h.helmBin, cmd)
-	if err != nil {
-		log.Error().Print("Error installing helm chart", field.M{"output": out})
-		return err
-	}
-	log.Debug().Print("Result", field.M{"output": out})
-	return nil
+	if !dryRun {
+		out, err := RunCmdWithTimeout(ctx, h.helmBin, cmd)
+		if err != nil {
+			log.Error().Print("Error installing helm chart", field.M{"output": out})
+			return "", err
+		}
+		log.Debug().Print("Result", field.M{"output": out})
+	} else {
+		cmd = append(cmd, "--dry-run")
+		log.Info().Print("Helm dry-run. Executing to capture output:")
+		out, err := RunCmdWithTimeout(ctx, h.helmBin, cmd)
+		if err != nil {
+			log.Error().Print("Error capturing output during Helm dry-run", field.M{"output": out, "error": err})
+			return "", err
+		}
+		log.Info().Print("Helm dry-run output:", field.M{"command": h.helmBin, "args": cmd, "output": out})
```

Comment:
let's make it `debug`. I don't think we should always print the dry run output.

Reply:
Done